rcl/test/rcl/test_timer.cpp

```diff
@@ -988,6 +988,91 @@ TEST_F(TestPreInitTimer, test_timer_get_period) {
   rcl_reset_error();
 }
 
+TEST_F(TestPreInitTimer, test_timer_info) {
+  int64_t next_call_start = 0;
+  int64_t old_period = 0;
+  times_called = 0;
+  rcl_timer_call_info_t call_info;
+  call_info.actual_call_time = 0;
+  call_info.expected_call_time = 0;
+  int64_t period = RCL_MS_TO_NS(10);
```

Comment:
AFAIS, this test uses `RCL_ROS_CLOCK` but storage is never enabled, then `now` returns system time. So we might see the timer overrun here already, and that could be flaky especially for windows CI? in that case, we would want to increase the period like `RCL_MS_TO_NS(100)`.

Reply:
I do not really know the platform spec for CI, but having 10 ms period could be flaky as far as i experienced... currently CI jobs seem okay. so we can go with `10 ms`, but i would use `50 ms` or `100 ms` to avoid meeting the flaky failure in the CI.